public/app/plugins/datasource/postgres/configuration/useAutoDetectFeatures.ts

```diff
@@ -38,7 +37,7 @@ export function useAutoDetectFeatures({ props, setVersionOptions }: Options) {
         // This is needed or else we get an error when we try to save the datasource.
         updateDatasourcePluginOption({ options, onOptionsChange }, 'version', result.datasource.version);
       } else {
-        const datasource = await getDatasourceSrv().loadDatasource(options.name);
+        const datasource = await getDataSourceSrv().get(options.name);
```

Comment:
the method `loadDatasource` is not exported on the data-type here, so i had to switch to `.get`. i checked the source code, and `.get()` will do some steps and then call `.loadDatasource`, so the change should be fine.

i also verified that below, when it calls `datasource.getVersion();`, the call is received by the postgres datasource plugin.